src/ray/raylet/node_manager.cc

```diff
@@ -314,7 +314,14 @@ NodeManager::NodeManager(instrumented_io_context &io_service, const NodeID &self
   cluster_resource_scheduler_ =
       std::shared_ptr<ClusterResourceScheduler>(new ClusterResourceScheduler(
           self_node_id_.Binary(), local_resources.GetTotalResources().GetResourceMap(),
-          *gcs_client_, [this]() { return object_manager_.GetUsedMemory(); },
+          *gcs_client_,
+          [this]() {
```

Comment:
Does this memory reporting get used by other components besides the autoscaler, and if so, how does the new flag affect those?

Reply:
Should be just autoscaler.